src/api/public/apidocs-new/paths/search_published_binary_id.yaml

```diff
@@ -0,0 +1,168 @@
+get:
+  summary: Search for currenlty available binaries in the publish area.
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: query
+      name: match
+      schema:
+        type: string
+      required: true
+      description: XPath expression.
+      example: project='home:foo'+and+name='bar'
+    - $ref: '../components/parameters/limit.yaml'
+    - $ref: '../components/parameters/withdownloadurl.yaml'
+  responses:
+    '200':
+      description: OK. The request has succeeded.
+      content:
```

Comment:
@eduardoj wouldnt it make sense to move those long responses to separate files?

Reply:
Done.